dist/libs/key_events.js

```diff
@@ -14,15 +14,17 @@ var _keycode2 = _interopRequireDefault(_keycode);
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
 var features = exports.features = {
-  FULLSCREEN: 1,
-  DOWN_PANEL: 2,
-  LEFT_PANEL: 3,
-  SHORTCUTS_HELP: 4,
-  ESCAPE: 5,
-  NEXT_STORY: 6,
-  PREV_STORY: 7,
-  SEARCH: 8,
-  DOWN_PANEL_IN_RIGHT: 9
+  FULLSCREEN: 'FULLSCREEN',
```

Comment:
I changed it to strings because it was hard to setup tests for that. Now all test passed so if it matters I can revert it if you want!?